homeassistant/components/intelliclima/const.py

```diff
@@ -0,0 +1,33 @@
+"""Constants for the IntelliClima integration."""
+
+import logging
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "intelliclima"
+
+# Update interval
+DEFAULT_SCAN_INTERVAL = 10  # seconds
```

Comment:
```suggestion
DEFAULT_SCAN_INTERVAL = timedelta(seconds=10)
```
This makes it easier to use this constant in other places, e.g. tests.